recipes/lenstool/meta.yaml

```
@@ -0,0 +1,58 @@
+{% set name = "lenstool" %}
+{% set version = "8.0.4" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://git-cral.univ-lyon1.fr/{{ name }}/{{ name }}/-/archive/{{ version }}/{{ name }}-{{ version }}.tar.gz
+  sha256: f14fb546a78ebbae3604739fe70089a98ca22668a85ba0d537fd9d63fc1c0786
+
+build:
+  number: 0
+  skip: true  # [win]
+  ignore_run_exports:
+    - python
+    - libgomp
+    - llvm-openmp
+
+requirements:
+  build:
+    - libtool  # [not win]
+    - {{ compiler('c') }}  # [not win]
+    - llvm-openmp  # [osx]
+    - libgomp      # [linux]
+    - make  # [not win]
+    - conda-verify
+  host:
+    - python
+    - pip
+    - cfitsio
+    - wcslib
+    - gsl
+    - libopenblas  # [linux]
```

Would a generic BLAS be ok here?
```suggestion
    - libblas  # [linux]
```